asmcli/lib/parse.sh

```diff
@@ -148,6 +143,8 @@ parse_args() {
         shift 1
         ;;
       --only_validate | --only-validate)
+        warn "As of version 1.11 the --only_validate flag is deprecated and will be ignored."
```

Comment:
I think "In version 1.11" is a bit more clear than "As of", since the latter implies that it might be already released.

Reply:
Done